src/bin/generate_and_query/main.rs

```diff
@@ -0,0 +1,80 @@
+use point_viewer::octree::build_octree;
+use point_viewer::read_write::{Encoding, NodeWriter, OpenMode, RawNodeWriter, S2Splitter};
+use random_points::{Batched, RandomPointsOnEarth};
+use std::path::PathBuf;
+use std::time::Instant;
+use structopt::StructOpt;
+
+mod random_points;
+
+#[derive(StructOpt, Debug)]
+struct CommandlineArguments {
+    /// Output directory to write the octree into.
```

Comment:
maybe change to " write the point cloud" (as s2 and octree are subdirectories of that)